common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -111,61 +111,11 @@ trait ABTestSwitches {
 
   Switch(
     ABTests,
-    "ab-sleeve-notes-new-email-variant",
-    "Assign some of the new sleeve notes subscribers to receive the new email",
-    owners = Seq(Owner.withGithub("lmath")),
-    safeState = Off,
-    sellByDate = new LocalDate(2017, 6, 1),
-    exposeClientSide = true
-  )
-
-  Switch(
-    ABTests,
-    "ab-sleeve-notes-legacy-email-variant",
-    "Assign some of the new sleeve notes subscribers to receive the old email",
-    owners = Seq(Owner.withGithub("lmath")),
-    safeState = Off,
-    sellByDate = new LocalDate(2017, 6, 1),
-    exposeClientSide = true
-  )
-
-  Switch(
-    ABTests,
-    "ab-the-long-read-email-variants",
```

Comment:
If we are deleting this test, do we need to also delete the aliases from `EmailNewsletters.scala`, and update the [interactive](https://github.com/guardian/interactive-email-signups/tree/master/the-long-read)?